docs/StardustDocs/topics/operations/utils/Utils.md

```diff
@@ -0,0 +1,22 @@
+# Utility functions
+
+
+<web-summary>
+Overview of common utility operations in Kotlin Dataframe.
+</web-summary>
+
+<card-summary>
+Overview of common utility operations in Kotlin Dataframe.
+</card-summary>
+
+<link-summary>
+Overview of common utility operations in Kotlin Dataframe.
+</link-summary>
+
+Explore frequently used helpers for querying and transforming your data:
+
+- [`all`](all.md) — Check whether all rows satisfy a predicate. 
+- [`any`](any.md) — Check whether any row satisfies a predicate.
```

Comment:
we really miss `none {}` in DataFrame, it's just `!any {}` but present in the stdlib